keras_cv/layers/preprocessing/equalization.py

```diff
@@ -79,13 +86,15 @@ def build_mapping(histogram, step):
 
         return tf.cast(result, dtype)
 
-    def call(self, images):
+    def augment_image(self, image, transformation=None):
         # Assumes RGB for now.  Scales each channel independently
         # and then stacks the result.
         # TODO(lukewood): ideally this would be vectorized.
-        r = tf.map_fn(lambda x: self.equalize_channel(x, 0), images)
-        g = tf.map_fn(lambda x: self.equalize_channel(x, 1), images)
-        b = tf.map_fn(lambda x: self.equalize_channel(x, 2), images)
+        dtype = image.dtype
+        image = tf.cast(image, tf.int32)
+        r = self.equalize_channel(image, 0)
```

Comment:
This is done. However, currently, I am using a nested `tf.vectorized_map`. The warnings are as follows now:

```
WARNING:tensorflow:Using a while_loop for converting StridedSlice
WARNING:tensorflow:Using a while_loop for converting HistogramFixedWidth
WARNING:tensorflow:Using a while_loop for converting HistogramFixedWidth
WARNING:tensorflow:Using a while_loop for converting DynamicPartition
WARNING:tensorflow:Using a while_loop for converting DynamicStitch
```
I hope this isn't a problem

Reply:
@qlzh727 @LukeWood 
Is the above warning message safe to ignore? Does the end-user need to worry about anything regarding this? If it's 100% okay to ignore, then I think the warning message should not appear in the first place. Otherwise, it would be a bad UX.

Reply:
It means that particular ops is not supported yet hence falling to while loop 
This does not affect the result but drain some performance.

Well if there are too many fallbacks to while loop, i suspect we wont have any/substantial  vectorization gain, (I might be wrong :) )
May be because too many fallback latency.

May be a benchmark, needed. 
@innat

Reply:
We have talked about this also at https://github.com/keras-team/keras-cv/pull/146#issuecomment-1049352918 and we are also trying to improve the feedback on the root cause of the fallback with https://github.com/tensorflow/tensorflow/pull/55192

The main issue also is that `tf.vectorized_map` [is currently orphan](https://github.com/keras-team/keras-cv/pull/146#issuecomment-1068486128)

Reply:
So, we should really mute this errors.  BUT I don't think it is blocking for now.  So resolving for now, but this is a tricky issue because it falls on the TensorFlow side, not our side.